### PR TITLE
release 1.21.0

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.20.0] - 2023-09-06
```
Comment (minump) [inline CHANGELOG.md:9]:
```suggestion
## [1.21.0] - 2023-09-07
```

File: SECURITY.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Patches for **Rokwire Building Blocks** in this repository will only be applied
 
 | Version  | Supported          |
```
Comment (minump) [inline SECURITY.md:7]: Need to change version in events.yaml file as well.

**Review by minump — CHANGES_REQUESTED** [review]
Some changes requested,

File: SECURITY.md (hunk 2 above)
Comment (bingzhang) [reply SECURITY.md:7]: fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (bingzhang) [reply CHANGELOG.md]: fixed.

**Review by minump — APPROVED** [review]
Approved

File: Dockerfile [hunk 3]
```diff
@@ -16,6 +16,6 @@ FROM swaggerapi/swagger-ui:v4.13.2
 
 COPY appconfigservice/appconfig.yaml eventservice/events.yaml profileservice/profile.yaml loggingservice/logging.yaml /usr/share/nginx/html/app/
 
-ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
+ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
```
Comment (sandeep-ps) [inline Dockerfile]: The URLs for develop should be changed to api-dev.

Comment (sandeep-ps) [inline Dockerfile]: It looks like during the previous release, the environment variables got updated. Fixing it through the below suggestion.

```suggestion
ENV BASE_URL="/docs" \
    URLS="[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/content/doc/ui/', name: 'Content Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/core/doc/ui/', name: 'Core Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/gr/doc/ui/', name: 'Groups Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/lms/doc/ui/', name: 'Learning Management System (LMS) Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/notifications/api/doc/ui/', name: 'Notifications Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/polls/doc/ui/', name: 'Polls Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/rewards/doc/ui/', name: 'Rewards Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/wellness/doc/ui/', name: 'Wellness Building Block'}]"
```

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**